README.md

```diff
@@ -1,2 +1,201 @@
-# identity-oauth2-grant-mfa
-identity-oauth2-grant-mfa is a REST API that enforces an OAuth2 grant type according to a set of multi factor authentication steps.
+# WSO2 Identity Server - API Authentication Connector
+
+Supported Versions : 5.10.0, 5.11.0, 6.0.0, 6.1.0
+
+This connector has developed to provide an RESTful API interface for 
```

Comment:
lets add a diagram showing the overall flow and a brief intro.

Reply:
Ideally reading the readme anyone should understand completely what this connector is, how to deploy it and how to use it.